continuedev/continue
# Installation 092424

## Description

- Added VS Code and Jetbrains logos, following their respective use guidelines found here:

    https://code.visualstudio.com/brand

    https://www.jetbrains.com/company/brand/#company-brand-logos-and-icons-jetbrains-logos

- Minor copy edits
- Style changes: spacing of ordered lists accomplished via css instead of adding spaces
- Some formatting changes to try to improve visual appeal

## Checklist


## Screenshots

![image](https://github.com/user-attachments/assets/712928af-21a1-40c0-9400-52f8cf4c1c49)

![image](https://github.com/user-attachments/assets/369f4a26-483e-4916-a2f7-82465cb96f70)


## Testing

I checked out the rest of the doc site just to make sure the changes to unordered lists and list items didn't have any side effects and didn't see any.

I've only tested on a 3440x1440 monitor on Chrome on Windows 11.

To test:

- `cd docs` in the project root
- `npm start`
- navigate to 'Installation' and make sure it looks kosher.
